app/channel/template.hbs

```diff
@@ -15,6 +15,8 @@
       action="pageNumberClicked"}}
 
   {{/alt-channel-layout}}
+{{else if openEarsTemplate}}
```

Comment:
I'd suggest we match this variable to the name of the template to reduce confusion, so either we call the template something related to OEP or change this variable to something related to `showdetail-fullbleed`.